reference/platform-app-yaml.md

```diff
@@ -6,48 +6,57 @@ You control your application and the way it will be built and deployed on Platfo
 via a single configuration file `.platform.app.yaml` located at the root of your application
 folder inside your Git repository.
 
-Here is an example of a `.platform.app.yaml` file for Drupal:
+Here's an example of a minimalist `.platform.app.yaml` file for PHP:
 
-    # The unique identifier of the application.
-    name: front
-    # The language that will run your application.
-    type: php:5.6
-    # The way to build your application.
-    build:
-        flavor: drupal
-    # The way services are mapped within your application.
-    relationships:
-        database: "mysql:mysql"
-        solr: "solr:solr"
-        redis: "redis:redis"
-    # The way your application is exposed to the web.
-    web:
-        locations:
-            "/":
-                root: "public"
-                expires: -1
-                passthru: "/index.php"
-                index:
-                    - index.php
-                allow: true
-    # The size of the persistent disk size of your application in MB.
-    disk: 2048
-    # The volumes that are mounted under a writable shared resource.
-    mounts:
-        "/public/sites/default/files": "shared:files/files"
-        "/tmp": "shared:files/tmp"
-        "/private": "shared:files/private"
-    # The shell commands to run during the build or deployment process.
-    hooks:
-        # We run deploy hook after your application has been deployed and started.
-        deploy: |
-            cd public
-            drush -y updatedb
-    # The processes that are triggered on a schedule.
-    crons:
-        drupal:
-            spec: "*/20 * * * *"
-            cmd: "cd public ; drush core-cron"
+```yaml
+# The name of this app. Must be unique within a project.
+name: 'app'
+
+# The type key specifies the language and version for your application.
+type: 'php:7.0'
+
+# On PHP, there are multiple build flavors available. Pretty much everyone except
+# Drupal 7 users will want the composer flavor.
+build:
+  flavor: 'composer'
+
+# The relationships of the application with services or other applications.
+# The left-hand side is the name of the relationship as it will be exposed
+# to the application in the PLATFORM_RELATIONSHIPS variable. The right-hand
+# side is in the form `<service name>:<endpoint name>`.
+relationships:
+    database: 'mysqldb:mysql'
+
+# The configuration of app when it is exposed to the web.
+web:
+    locations:
+        '/':
+            # The public directory of the app, relative to its root.
+            root: 'web'
+            # The front-controller script to send non-static requests to.
+            passthru: '/app.php'
+        # Allow uploaded files to be served, but do not run scripts.
+        '/files'
+            scripts: false
+            allow: true
+
+# The size of the persistent disk of the application (in MB).
+disk: 2048
+
+# The mounts that will be performed when the package is deployed.
+mounts:
+    '/app/web/files': 'shared:files/cache'
```

Comment:
`cache` is probably not accurate? `shared:files/files` or `shared:files/public-files` would be clearer to me